### DISCUSSION TITLE
Custom next/image cache TTL overrides

### DISCUSSION THREAD

**Discussion by kaleabmelkie** [vercel/next.js#19180] [Ideas]
As per [the docs](https://nextjs.org/docs/basic-features/image-optimization#caching) , next/images use the upstream URL's Cache-Control maxAge (otherwise default to 60 seconds if the header is not found).

Is there a way (or an upcoming PR) to override the maxAge for the images (either individually or globally) to a custom value of our own?

_Example use case: Gravatar provides a 5 minutes TTL, but passing through next/image, is it possible to override that to a much higher maxAge?_

**Comment by leerob** [comment]
https://nextjs.org/docs/app/api-reference/components/image#minimumcachettl
